torchmdnet/models/output_modules.py

```diff
@@ -45,8 +44,11 @@ def pre_reduce(self, x, v, z, pos, batch):
         return
 
     def reduce(self, x, batch):
-        if not self.setup_for_compile:
-            is_capturing = x.is_cuda and is_current_stream_capturing()
+        if torch.jit.is_scripting():
+            # TorchScript doesn't support torch.cuda.is_current_stream_capturing()
+            self.dim_size = int(batch.max().item() + 1)
+        elif not self.setup_for_compile:
+            is_capturing = x.is_cuda and torch.cuda.is_current_stream_capturing()
```

Comment:
If I remember correctly, I introduced this because otherwise it was not possible to torch compile a model that was saved as torchscript (so for instance, one running via OpenMM-Torch). Are you checking for that?

Reply:
Hm no, this is not something covered in the test suite? I have not run any further tests other than the ones in tests/ and the benchmarks.

Reply:
I was actually referring to CUDA graphing. Sorry -.-
This is what the "is_cuda_graph_capturing" does and what is used in OpenMM-Torch. So a test like this but TorchScripting before graphing:
https://github.com/torchmd/torchmd-net/blob/75b16e625039a5e5e79c5e9d679c6c92ee274445/tests/test_model.py#L117-L161

BTW surprised only tensornet is graphable these days!